frontend/src/components/ShootWorkers/WorkerGroup.vue

```diff
@@ -55,91 +56,135 @@ export default {
       'volumeTypesByCloudProfileName',
       'machineImagesByCloudProfileName'
     ]),
-    machineTypes () {
-      return this.machineTypesByCloudProfileName({ cloudProfileName: this.cloudProfileName })
+    machineType () {
+      const machineTypes = this.machineTypesByCloudProfileName({ cloudProfileName: this.cloudProfileName })
+      const type = get(this.workerGroup, 'machine.type')
+      return find(machineTypes, ['name', type])
     },
-    volumeTypes () {
-      return this.volumeTypesByCloudProfileName({ cloudProfileName: this.cloudProfileName })
+    volumeType () {
+      const volumeTypes = this.volumeTypesByCloudProfileName({ cloudProfileName: this.cloudProfileName })
+      const type = get(this.workerGroup, 'volume.type')
+      return find(volumeTypes, ['name', type])
     },
-    machineImages () {
-      return this.machineImagesByCloudProfileName(this.cloudProfileName)
+    machineImage () {
+      const machineImages = this.machineImagesByCloudProfileName(this.cloudProfileName)
+      const { name, version } = get(this.workerGroup, 'machine.image', {})
+      return find(machineImages, { name, version })
     },
-    description () {
+    workerGroupDescriptions () {
```

Comment:
Why `workerGroupDescriptions` and not just `descriptions`? The name of the component is `WorkerGroup`!